pkg/user/verification_test.go

```diff
@@ -104,7 +104,7 @@ func TestVerification_Expiration(t *testing.T) {
 
 func TestVerification_IsExpired(t *testing.T) {
 	tim, _ := time.Parse(time.RFC3339, "2021-03-16T04:19:57.592Z")
-	tim2, _ := time.Parse(time.RFC3339, "2022-03-16T04:19:57.592Z")
+	tim2 := time.Now().Add(time.Hour * 24)
```

Comment:
ok for now but also remember this way:

```go
var timeNow = time.Now

func hoge() {
  now := timeNow()
}
```

```go
func Test_hoge() {
	timeNow = func () time.Time {
		return time.Time{...}
	}
	defer timeNow = time.Now

	// test
}
```

NOTE: `t.Parallel()` is not supported